app/crud/comprehensive_evaluation.py

```diff
@@ -0,0 +1,100 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from app.models.evaluation import ComprehensiveEvaluation
+from uuid import UUID
+from typing import List
+from sqlalchemy import select, delete
+from app.schemas.comprehensive_evaluation import (
+    ComprehensiveEvaluationCreate,
+    ComprehensiveEvaluationResponse,
+    ComprehensiveEvaluationUpdate,
+)
+from app.utils.to_snake_case import camel_to_snake
+
+
+async def create_comprehensive_evaluation(
+    db: AsyncSession,
+    user_id: UUID,
+    comprehensive_evaluation: ComprehensiveEvaluationCreate,
+) -> ComprehensiveEvaluationResponse:
+
+    db_comprehensive_evaluation = ComprehensiveEvaluation(
+        user_id=user_id,
+        overall_achievement_rate=comprehensive_evaluation.overallAchievementRate,
+        evaluation_text=comprehensive_evaluation.evaluationText,
+        strength_achievement_rate=comprehensive_evaluation.strengthAchievementRate,
+        strength_text=comprehensive_evaluation.strengthText,
+        improvement_achievement_rate=comprehensive_evaluation.improvementAchievementRate,
+    )
+
+    db.add(db_comprehensive_evaluation)
+    await db.commit()
+    await db.refresh(db_comprehensive_evaluation)
+    return db_comprehensive_evaluation
+
+
+async def read_comprehensive_evaluations(
+    db: AsyncSession, user_id: UUID
+) -> List[ComprehensiveEvaluationResponse]:
+    comprehensive_evaluations = await db.execute(
+        select(ComprehensiveEvaluation).where(
+            ComprehensiveEvaluation.user_id == user_id
+        )
+    )
+    return comprehensive_evaluations.scalars().all()
+
+
+async def read_comprehensive_evaluation(
+    db: AsyncSession, user_id: UUID, comprehensive_evaluation_id: UUID
+) -> ComprehensiveEvaluationResponse:
+    comprehensive_evaluation = await db.execute(
+        select(ComprehensiveEvaluation).where(
+            ComprehensiveEvaluation.id == comprehensive_evaluation_id,
+            ComprehensiveEvaluation.user_id == user_id,
+        )
+    )
+
+    result = comprehensive_evaluation.scalars().first()
+
+    return result
+
+
+async def update_comprehensive_evaluation(
+    db: AsyncSession,
+    user_id: UUID,
+    comprehensive_evaluation_id: UUID,
+    comprehensive_evaluation: ComprehensiveEvaluationUpdate,
+) -> ComprehensiveEvaluationResponse:
+    db_comprehensive_evaluation = await read_comprehensive_evaluation(
+        db, user_id, comprehensive_evaluation_id
+    )
+
+    update_data = comprehensive_evaluation.model_dump(exclude_unset=True)
+
+    converted_data = {}
+
+    # 모든 키를 카멜케이스에서 스네이크케이스로 변환
+    for key, value in update_data.items():
+        snake_key = camel_to_snake(key)
+        converted_data[snake_key] = value
+
+    # 변환된 키-값 쌍으로 ORM 객체 업데이트
+    for key, value in converted_data.items():
+
+        setattr(db_comprehensive_evaluation, key, value)
```

Comment:
지금 코드 구현해보고 있는데 아래처럼 짜도 동작하지 않을까 싶어 코멘트 추가합니당.

```suggestion

    for key, value in update_data.items():
        snake_key = to_snake_case(key)
        setattr(db_memo, snake_key, value)
```

Reply:
@GideokKim 
오 그렇네요!! 이 부분도 반영해서 수정 완료 했습니다!